src/System.Diagnostics.Tracing/documentation/EventCounterTutorial.md

```diff
@@ -47,9 +47,7 @@ So, with that, we logged the metric to the `EventCounter`, but unless we can act
 
 There is an extra keyword that you will need to specify the turn on the EventCounters.
 
-```
-PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect
-```
+<code>PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:**EventCounterIntervalSec=1** collect</code>
```

Comment:
sorry @karelz, i missed this review earlier. so, you're saying that I should that all \* in that command are part of the syntax? If that's the case, I'll revert to the original.